example/rocketmq/src/main/java/org/apache/iotdb/rocketmq/RocketMQConsumer.java

```diff
@@ -120,7 +120,7 @@ public void prepareConsume() throws MQClientException {
             new String(msg.getBody())));
         try {
           insert(new String(msg.getBody()));
-        } catch (IoTDBConnectionException e) {
+        } catch (Exception e) {
           logger.error(e.getMessage());
```

Comment:
it is better to throw MQClientException

Reply:
The  MessageListenerOrderly interface doesn't allow to throw an exception